plugin.go

```diff
@@ -376,15 +379,26 @@ func (p *ormPlugin) generateDefaultHandlers(file *generator.FileDescriptor) {
 		p.gormPkgAlias = pkgGORM.Name()
 		pkgContext := p.NewImport("context")
 		pkgContext.Use()
-		p.generateCreateHandler(file, message)
-		p.generateReadHandler(file, message)
-		p.generateUpdateHandler(file, message)
-		p.generateDeleteHandler(file, message)
+		p.generateCreateHandler(message)
+		p.generateReadHandler(message)
+		p.generateUpdateHandler(message)
+		p.generateDeleteHandler(message)
 		// p.generateListHandler(file, service, method)
 	}
 }
 
-func (p *ormPlugin) generateCreateHandler(file *generator.FileDescriptor, message *generator.Descriptor) {
+func (p *ormPlugin) GetOption(message *generator.Descriptor) *gorm.GormMessageOptions {
```

Comment:
Message options are the most common, but is "GetOption" descriptive enough, given the number of possible option types?

Reply:
that's a good call.. I keep forgetting Go doesn't have method overloading